app/assets/v2/js/pages/bounty_details.js

```diff
@@ -940,7 +949,7 @@ var build_detail_page = function(result) {
     $('#funder_notif_info').append('\
         <span class="bounty-notification ml-2">\
         <i class="far fa-bell"></i>\
-        Ready to Pay? Set Your Metamask to this address!\
+        Ready to Pay? Set Your wallet to this address!\
```

Comment:
Some places were changed to say "wallet" instead of "Metamask", but there are a lot of other places where "MetaMask" is still used. I didn't want to make the PR so big, so I just left them as they were.